docs/src/common.md

```diff
@@ -151,36 +156,41 @@ H(q0) == J0
 
 Following the `DifferentialEquations.jl`
 [tutorial](http://docs.juliadiffeq.org/latest/tutorials/ode_example.html),
-we define an `ODEProblem` for the integration;
-`TaylorIntegration.jl` can be used via its common interface bindings with `DiffEqBase.jl`; both packages need to be loaded explicitly.
+we define an `ODEProblem` for the integration; `TaylorIntegration` can be used
+via its common interface bindings with `DiffEqBase.jl`; both packages need to
+be loaded explicitly.
 ```@example common
-tspan = (0.0, 1000.0)
+tspan = (0.0, 2000.0)
 p = [μ]
 
-using TaylorIntegration, DiffEqBase
-prob = ODEProblem(f, q0, tspan, p)
+using DiffEqBase
+prob = ODEProblem(pcr3bp!, q0, tspan, p)
 ```
 
-We solve `prob` using a 25-th order Taylor method, with a local absolute tolerance ``\epsilon_\mathrm{tol} = 10^{-20}``.
+We solve `prob` using a 25-th order Taylor method, with a local absolute tolerance ``\epsilon_\mathrm{tol} = 10^{-15}``.
 ```@example common
-solT = solve(prob, TaylorMethod(25), abstol=1e-20);
+solT = solve(prob, TaylorMethod(25), abstol=1e-15);
 ```
 
 As mentioned above, we load `OrdinaryDiffEq` in order to solve the same problem `prob`
-now with the `Vern9` method, which the `DifferentialEquations.jl`
+now with the `Vern9` method, which the `DifferentialEquations`
 [documentation](http://docs.juliadiffeq.org/latest/solvers/ode_solve.html#Non-Stiff-Problems-1)
```

Comment:
The link to DifferentialEquations.jl docs seems broken; I think the correct one is https://diffeq.sciml.ai/stable/solvers/ode_solve/#Non-Stiff-Problems

Reply:
Thanks, fixed!